lib/sudo-cutils/src/lib.rs

```diff
@@ -73,7 +75,7 @@ pub unsafe fn os_string_from_ptr(ptr: *const libc::c_char) -> OsString {
 /// Create a C string copy of a Rust string copy, allocated by libc::malloc()
 ///
 /// The returned pointer **must** be cleaned up via a call to `libc::free`.
-pub fn into_leaky_cstring(s: &str) -> *const libc::c_char {
+pub fn into_leaky_cstring(s: &[u8]) -> *const libc::c_char {
```

Comment:
I think this function can use a new name (and description) since its original inception, basically it is now a 'leaky_copy_for_ffi' or something like that.